specs/ClearBrowsingData.md

```diff
@@ -100,79 +100,82 @@ interface ICoreWebView2ClearBrowsingDataCompletedHandler;
 
 /// Specifies the datatype for the
 /// `ICoreWebView2Profile::ClearBrowsingData` method.
-[v1_enum] 
+[v1_enum]
 typedef enum COREWEBVIEW2_BROWSING_DATA_KINDS {
-  /// Specifies data stored by the AppCache DOM feature.
```

Comment:
Please don't update the indenting while updating the enum values. It makes it difficult to see what has changed when using the github diff.

Reply:
Will do.